TGM/src/main/java/network/warzone/tgm/modules/DeathMessageModule.java

```diff
@@ -50,29 +50,45 @@ public void onTGMDeath(TGMPlayerDeathEvent event) {
                 if (weapon != null && weapon.getType().equals(Material.BOW))
                     message = playerTeam.getColor() + module.getPlayerName() + ChatColor.GRAY + " was shot into the void by " +
                             killerTeam.getColor() + module.getKillerName() + ChatColor.GRAY;
-                else
-                    if (!module.getPlayerName().equals(module.getKillerName())) {
-                        message = playerTeam.getColor() + module.getPlayerName() + ChatColor.GRAY + " was thrown into the void by " +
-                                killerTeam.getColor() + module.getKillerName() + ChatColor.GRAY + " using " +
-                                itemToString(weapon);
-                    } else {
-                        message = playerTeam.getColor() + module.getPlayerName() + ChatColor.GRAY + " fell into the void";
-                    }
+                else if (!module.getPlayerName().equals(module.getKillerName())) {
+                    message = playerTeam.getColor() + module.getPlayerName() + ChatColor.GRAY + " was thrown into the void by " +
+                            killerTeam.getColor() + module.getKillerName() + ChatColor.GRAY + " using " +
+                            itemToString(weapon);
+                } else {
+                    message = playerTeam.getColor() + module.getPlayerName() + ChatColor.GRAY + " fell into the void";
+                }
+
+            } else if (cause.equals(DamageCause.DROWNING)) {
+                if (!module.getPlayerName().equals(module.getKillerName())) {
+                    message = playerTeam.getColor() + module.getPlayerName() + ChatColor.GRAY + " drowned thanks too " + killerTeam.getColor() + module.getKillerName() + ChatColor.GRAY;
```

Comment:
Grammar error, might want to fix that

Reply:
whats wrong with it? lucas approved it lol

Reply:
"drowned thanks too" isn't correct, it's supposed to be drowned thanks to